src/files/modals/share-modal/ShareModal.js

```diff
@@ -9,8 +10,13 @@ import { Modal, ModalActions, ModalBody } from '../../../components/modal/Modal.
 const ShareModal = ({ t, tReady, onLeave, link, className, ...props }) => (
   <Modal {...props} className={className} onCancel={onLeave} >
     <ModalBody title={t('shareModal.title')} Icon={ShareIcon}>
```

Comment:
nit: if we show QR code, we should not waste vertical space for Icon:

> ![image](https://github.com/user-attachments/assets/17598e77-eb6a-4d4f-97a2-1a187f5a476e)

Then, thanks to reclaimed space, you can make QR code bigger.

Reply:
Edited as suggested

<img width="621" alt="image" src="https://github.com/user-attachments/assets/4266640e-9201-47d5-be79-374b19f4afb8" />